src/main/groovy/org/transmartproject/batch/batchartifacts/DbMetadataBasedBoundsValidator.groovy

```diff
@@ -83,14 +82,24 @@ class DbMetadataBasedBoundsValidator implements Validator {
         }
     }
 
+    private String getIdentifier(DatabaseMetaData meta, String name) {
+        if (meta.storesUpperCaseIdentifiers()) {
+            name.toUpperCase()
+        } else if (meta.storesLowerCaseIdentifiers()) {
+            name.toLowerCase()
+        } else {
+            name
+        }
+    }
+
     private Map processColumnSpecification(DatabaseMetaData meta,
                                            ColumnSpecification spec) {
-        // TODO: check which character oracle uses to escape the like expressions
+
         ResultSet rs = meta.getColumns(
                 null,
-                StringUtils.escapeForLike(spec.schema),
-                StringUtils.escapeForLike(spec.table),
-                StringUtils.escapeForLike(spec.column),)
+                getIdentifier(meta, spec.schema),
+                getIdentifier(meta, spec.table),
+                getIdentifier(meta, spec.column))
```

Comment:
The javadocs say this is a `LIKE` expression. Therefore, the `_` and `%` must be escaped.